cmd/scheduler-estimator/app/options/options.go

```diff
@@ -35,8 +38,10 @@ type Options struct {
 	Master      string
 	ClusterName string
 	// BindAddress is the IP address on which to listen for the --secure-port port.
+	// Deprecated: Use HealthProbeBindAddress ＆ MetricsBindAddress instead. And will be removed in release 1.12+.
```

Comment:
how about
```suggestion
	// Deprecated: To specify the TCP addresses for serving health probes, use HealthProbeBindAddress instead. To specify the TCP addresses for serving prometheus metrics, use MetricsBindAddress instead. And will be removed in release 1.12+.
```
or
```suggestion
	// Deprecated: Use HealthProbeBindAddress for health probes and MetricsBindAddress for Prometheus metrics. And will be removed in release 1.12+.
```

Reply:
Sounds good! Thanks for the comment. I will update them with the first description.